### ISSUE TITLE
Difference of stratified thermal storage to capacity model or fully mixed model not clear

### ISSUE THREAD

**Issue by jnnr** [oemof/oemof-thermal#100]
There is no clear data or figure that helps to choose if one wants to use the stratified thermal storage or a simpler model for thermal storages. The plots coming out of examples/stratified_thermal_storage.py give an idea about the difference in the concept, but not about the consequences of choosing this component.

Running a typical model with a simple vs. stratified thermal storage for a whole year would help to quantify the differences.

**Comment by c-moeller** [comment]
@jakob-wo this topic could be related to the validation of the thermal storage

I'm going to shift this to v0.0.4 for now due to many other open issues in v0.0.3

**Comment by jakob-wo** [comment]
@c-moeller I do not see a connection to the validation (with measurement data). It rather seems to be about new or adapted examples that show the differences between two models.

**Comment by jnnr** [comment]
I agree with @c-moeller. It is a main motivation of validation to be able to tell if a simple model is good enough or if the next level of complexity is necessary. If two models yield pretty similar results and both fit the data, you would always choose the simpler one.

**Comment by jakob-wo** [comment]
I understand your interpretation of _validation_ and I agree that a model comparision is essential but that is not what I am working on right now.
The comparision between "simple" and "complex" model is part of the docu already: [Stratified thermal storage (see Fig. 2)](https://oemof-thermal.readthedocs.io/en/latest/stratified_thermal_storage.html).
I agree that this part should be improved to highlight in what cases the "more complex" model can be an advantage. To me that is an improvement of the docu and perhaps of the examples.

**Comment by c-moeller** [comment]
Could be completed within the pv-heatpump simulations in the GRECO project.
